test/integration/github.test.js

```diff
@@ -10,25 +10,25 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-const Octokit = require('@octokit/rest');
+const { Octokit } = require('@octokit/rest');
 
 if (!process.env.TEST_ONE_PAC) throw new Error('environment variable `TEST_ONE_PAC` not present, aborting. This is required for integration tests against GitHub.com.');
 if (!process.env.TEST_TWO_PAC) throw new Error('environment variable `TEST_TWO_PAC` not present, aborting. This is required for integration tests against GitHub.com.');
-if (!process.env.TEST_FOUR_PAC) throw new Error('environment variable `TEST_FOUR_PAC` not present, aborting. This is required for integration tests against GitHub.com.');
-if (!process.env.TEST_MAJ_PAC) throw new Error('environment variable `TEST_MAJ_PAC` not present, aborting. This is required for integration tests against GitHub.com.');
-if (!process.env.TEST_MAJ583_PAC) throw new Error('environment variable `TEST_MAJ583_PAC` not present, aborting. This is required for integration tests against GitHub.com.');
+if (!process.env.TEST_THREE_PAC) throw new Error('environment variable `TEST_THREE_PAC` not present, aborting. This is required for integration tests against GitHub.com.');
+// if (!process.env.TEST_FOUR_PAC) throw new Error('environment variable `TEST_FOUR_PAC` not present, aborting. This is required for integration tests against GitHub.com.');
+// if (!process.env.TEST_FIVE_PAC) throw new Error('environment variable `TEST_FIVE_PAC` not present, aborting. This is required for integration tests against GitHub.com.');
```

Comment:
delete?

Reply:
leaving integration tests mostly alone now since they can't be run at the moment and i'd like it to be easy to look back at it